src/main/java/com/example/springlabs/StoreController.java

```diff
@@ -39,20 +41,21 @@ public String newCategory(Model model) {
 
     @PostMapping("/createCategory")
     public String createCategory(
-            @RequestParam("categoryId") String idSt,
             @RequestParam("categoryName") String name,
             @RequestParam("categorySelect") String parentCategoryIdSt,
             Model model
     ) {
-        int id = Integer.parseInt(idSt);
-
         if (parentCategoryIdSt.isEmpty()) {
-            Category category = new Category(id, name);
+            Category category = new Category(name);
             model.addAttribute("categories", categoryService.addCategory(category));
         } else {
             int parentCategoryId = Integer.parseInt(parentCategoryIdSt);
```

Comment:
use long? Also, delete stiil use int, please fix it

Reply:
It is more related to Issue #6 than this. I only made changes for application to compile.